rust/src/wrapper/worksheet.rs

```diff
@@ -270,7 +270,7 @@ impl Worksheet {
     ///
     /// TODO: example omitted
     #[wasm_bindgen(js_name = "setColumnWidthPixels", skip_jsdoc)]
-    pub fn set_column_width_pixels(&self, col: xlsx::ColNum, width: u16) -> WasmResult<Worksheet> {
+    pub fn set_column_width_pixels(&self, col: xlsx::ColNum, width: u32) -> WasmResult<Worksheet> {
```

Comment:
There are some slight changes to the API, but it doesn't affect JavaScript API.
https://github.com/jmcnamara/rust_xlsxwriter/commit/c826ba0fdda9edbc7619db9fbdbfd19c034936fe